Paintroid/src/main/java/org/catrobat/paintroid/model/LayerModel.kt

```diff
@@ -50,8 +55,12 @@ class LayerModel : LayerContracts.Model {
         layers[position] = layer
     }
 
-    override fun removeLayerAt(position: Int) {
+    @SuppressWarnings("SwallowedException", "TooGenericExceptionCaught")
+    override fun removeLayerAt(position: Int): Boolean = try {
         layers.removeAt(position)
+        true
+    } catch (e: IndexOutOfBoundsException) {
+        false
     }
 
     companion object {
```

Comment:
just noticed, could you please also remove the `@JvmStatic` annotations from the methods in the companion object, they're no longer required, they were added initially when some of the usages were in java.